EIPS/eip-7870.md

```diff
@@ -0,0 +1,152 @@
+---
+eip: 7870
+title: Hardware and Bandwidth Recommendations for Validators and Full Nodes
+description: System recommendations for Validators and Full nodes
+author: Parithosh Jayanthi (@parithosh), Kevaundray Wedderburn (@kevaundray), Josh Rudolf (@jrudolf), Dankrad Feist (@dankrad), Justin Traglia (@jtraglia), Ignacio Hagopian (@jsign), George Kadianakis (@asn-d6)
+discussions-to: https://ethereum-magicians.org/t/hardware-and-bandwidth-recommendations-for-full-nodes-and-validators/22675
+status: Draft
+type: Informational
+created: 2025-01-26
+---
+
+## Abstract
+
+This proposal specifies hardware and bandwidth recommendations for full nodes and validators. Validators are split into attesters and proposers. The resource-intensive aspect of block proposal lies in creating the block and broadcasting the data required for attesters to validate it. We use the term "local block builder" to further emphasize this separation.
+
+We note that you may be able to run a client with less than the recommended specifications, however benchmarks and decision-making will be made with respect to these recommendations.
+
+## Motivation
+
+Clear system specifications are crucial for:
+
+- Ensuring meaningful benchmark comparisons across different implementations
+- Enabling informed decision-making about protocol upgrades and their resource usage implications
+- Providing clear guidance for node operators with respect to the future
+
+Without a shared understanding of target hardware specifications:
+
+- Benchmark results lose significance due to inconsistent testing environments
+- Decision-making becomes challenging for implementation choices, as performance characteristics are heavily hardware-dependent
+- Network participants lack clear guidance for hardware investments
+
+## Specification
+
+### Recommended hardware and bandwidth specifications
```

Comment:
Maybe include a date here, so if users stumble upon this EIP they will know that this is recent, and not recommended specifications of 3 years ago.

Reply:
We should keep this EIP up to date or explicitly replace it with another in the future